In [1]:
import numpy as np
import pandas as pd

df = pd.read_csv('Pengunjung_Mall.csv')
df

ID_Pelanggan  Gender  Usia  Pendapatan_Tahunan_Ribuan_USD  \
0               1    Pria    19                             15   
1               2    Pria    21                             15   
2               3  Wanita    20                             16   
3               4  Wanita    23                             16   
4               5  Wanita    31                             17   
..            ...     ...   ...                            ...   
195           196  Wanita    35                            120   
196           197  Wanita    45                            126   
197           198    Pria    32                            126   
198           199    Pria    32                            137   
199           200    Pria    30                            137   

     Pengeluaran_USD  
0                 39  
1                 81  
2                  6  
3                 77  
4                 40  
..               ...  
195               79  
196               28  
197               74  
198               18  
199               83  

[200 rows x 5 columns]

<h2>Skenario Analisa Data Pengunjung Mall</h2>
<p>Memuat jumlah pelanggan, jenis kelamin, usia, pendapatan, dan pengeluaran</p>
<ol>
    <li>Jumlah Pelanggan adalah 200 orang</li>
    <li>Gender Pelanggan adalah Pria dan Wanita</li>
    <li>Usia Pelanggan Pria Termuda adalah 18 Tahun</li>
    <li>Usia Pelanggan Pria Tertua adalah 70 Tahun</li>
    <li>Usia Pelanggan Wanita Termuda adalah 18 Tahun</li>
    <li>Usia Pelanggan Wanita Tertua adalah 68 Tahun</li>
    <li>Pendapatan Tahunan Terkecil (Ribuan) Pelanggan Pria : 15 USD</li>
    <li>Pendapatan Tahunan Terbesar (Ribuan) Pelanggan Pria : 137 USD</li>
    <li>Pendapatan Tahunan Terkecil (Ribuan) Pelanggan Wanita : 16 USD</li>
    <li>Pendapatan Tahunan Terbesar (Ribuan) Pelanggan wanita : 126 USD</li>
    <li>Pengeluaran Terkecil Pelanggan Pria : 1 USD</li>
    <li>Pengeluaran Terbesar Pelanggan Pria : 97 USD</li>
    <li>Pengeluaran Terkecil Pelanggan Wanita : 5 USD</li>
    <li>Pengeluaran Terbesar Pelanggan Wanita : 99 USD</li>
</ol>